aleph/views/entities_api.py

```diff
@@ -33,6 +35,20 @@ def index():
     return EntitySerializer.jsonify_result(result)
 
 
+@blueprint.route('/api/2/search/export/<any(csv, excel):format>', methods=['GET'])  # noqa
+@blueprint.route('/api/2/entities/export/<any(csv, excel):format>', methods=['GET'])  # noqa
+def export(format):
+    parser = SearchQueryParser(request.args, request.authz)
+    result = EntitiesQuery.handle(request, parser=parser)
+    results = result.to_dict(serializer=EntitySerializer)['results']
```

Comment:
At the moment, this will return only the first `limit` results, but max out at 9999. I think this is unexpected behaviour, we should return a fixed number of max results (e.g. 1000?) and disable the option in the UI if the result total is larger than that. 

/cc @dkhurshudian